content/applications/hr/referrals.rst

```diff
@@ -193,15 +195,15 @@ To share the job position using Facebook, click the :guilabel:`Share Now` button
 with the link populated in the main body of the new post. If the user is *not* already logged in, a
 log-in screen loads instead, prompting the user to log-in to Facebook first.
 
-Type in any additional information to add to the post, in the space directly below your name. If
-desired, :guilabel:`tag` users, :guilabel:`check-in` at a location, or add a :guilabel:`status` to
-the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag to the
-post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location from
-the pop-up of predetermined locations based on your current location. To add a :guilabel:`status`,
-click the :guilabel:`Feeling/Activity` icon, then select either the feeling or the activity type,
-and lastly, if an activity type was selected, select the specific activity.
+Type in any additional information to add to the post, in the space directly below the user's name.
+If desired, :guilabel:`tag` users, :guilabel:`check-in` at a location, or add a :guilabel:`status`
+to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag in
+the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location
+from the pop-up of predetermined locations based on the user's current location. To add a
+:guilabel:`status`, click the :guilabel:`Feeling/Activity` icon, and select either the feeling or
+the activity type, and lastly, if an activity type was selected, select the specific activity.
```

Comment:
I don't believe 'tag', 'check-in', nor 'status' need a GUI, since that is not how those fields appear on the UI

Can you also break up this large paragraph, please?

Describe the check in icon

mention "window" (or "form" when applicable) after every mention of "pop-up"

can you describe the look of the 'feeling/activity' icon?